tests/apps/core/test_error_view.py

```diff
@@ -71,9 +84,20 @@ def test_error_view_handler_with_unsupported_status_code(self):
         the 500 error view should be displayed
         """
         page = create_page("Test", "richie/single_column.html", "en", published=True)
+        page.set_as_homepage()
+
         request = RequestFactory().get("/")
         request.current_page = page
         request.user = AnonymousUser()
+
         with self.assertTemplateUsed("richie/error.html"):
             response = error.error_view_handler(request, Exception, 405)
             self.assertContains(response, "500 - Server bad request", status_code=500)
+
+    def test_error_view_handler_raises_error_when_no_homepage(self):
+        """
+        As error_view_handler set context['current_page'] to the homepage site,
```

Comment:
Or make it work when there is no home page?

Reply:
Yep. Can we assume to pass None if the exception `NoHomeFound` is raised ? I guess if a site has no homepage, there is also no plugins used?

Reply:
Yes I agree with this take. Maybe add a test on the sitemap plugin that was problematic to be sure it behaves when current_page is set to None.